### DISCUSSION TITLE
Welcome to kiwix-android Discussions!

### DISCUSSION THREAD

**Discussion by gouri-panda** [kiwix/kiwix-android#2750] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cdhiraj40** [comment]
Hello fellow contributors, I am Dhiraj and I have started contributing to kiwix-android some days ago and hopefully will be helping the organization more. I would love to be part of future discussions, looking forward to learning and sharing my thoughts :)

**Comment by Vinay-Khanagavi** [comment]
![image](https://github.com/user-attachments/assets/02c9bcd1-b682-42c1-b326-24d2856c4691)

I am getting this error in mainactivity!? any solution

**Comment by ABHISHEKCHHIKARA** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

I was thinking of 1 issue that hasn't been resolved properly for a long time , the issue with the library feature, recently it allowed me to open one .zim file by picking up , I think that's the only thing it managed to solve yet , but in its own local directory in sd card , it didn't pick up any files it downloaded just a moment ago , also , earlier today I installed the new version from playstore after uninstalling the previous git hub one ( sd card was disconnected so it didn't wipe data at that moment ) later on I started downloading khan academy and gutenberg project in external directory , I left it there for it download , to my surprise I came back around 5 hours later , the download was stopped due to some network error and it wiped all of the previously downloaded files in the directory . This new changelog has been a literal disaster. Atleast the previous ones didn't wipe anything out . 
I hope it gets fixed lately
